data/korath/korath missions.txt

```diff
@@ -577,3 +577,19 @@ mission "Efreti: Distress Call 8"
 			`	You return to your ship and try plugging the chip into your systems again, which now displays two files: one in the Korath language, and one in your own. You open the file in your language, and it reads: "The <EDC-ship> is a ship of grief, bringing great pain in memories that sing for generations. Love is a void in the souls of many. Know all that <first> <last> is a true Efret by heart, if not by skin. The <EDC-ship> is theirs, to save lives and keep memories."`
 			`	You carefully store the chip in a safe place.`
 
+
+mission "Exile: Boarding Worldship Korath"
+	boarding
+	minor
+	destination "Earth"
+	source
+		government "Korath" "Korath (Civilian)"
+		category "Superheavy"
+		outfits "Afterburner (Stellar Class)"
+	on offer
+		conversation
+			`Docking with the gargantuan vessel allows you a front-row seat to witness the sheer scale of this ship. Your sensors detect tens of thousands of life signs aboard, most of which read as likely intelligent, with many thousands more minor life signs alongside them.`
```

Comment:
> Your sensors detect tens of thousands of life signs aboard

Minor detail, but the required bunks is 1940 of 31920. Thus, the number of life signs is likely to be less than tens (plural) of thousands.

Reply:
We could just say that it's detecting hatchlings

Reply:
Yeah, once it is able to be captured it will be better to change that to thousands or something closer to whatever the average number of crew will be, but right now it's assuming the ships are all full of crew and citizens since for a city ship, you would assume they wouldn't leave empty rooms

Reply:
I would not plan on it ever being capturable.